adapter/pkg/config/log_types.go

```diff
@@ -30,8 +30,9 @@ type accessLog struct {
 
 // LogConfig represents the configurations related to adapter logs and envoy access logs.
 type LogConfig struct {
-	Logfile  string
-	LogLevel string
+	Logfile          string
+	LogLevel         string
+	LogFormalization string
```

Comment:
Should be changed as `LogFormat` - (values: "JSON", "TEXT")

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2633/commits/c3087100180167139f73a7fb351a91359c5d9bbf